holoviews/plotting/bokeh/widgets.py

```diff
@@ -1,12 +1,178 @@
+from __future__ import unicode_literals
+
 import json
+from functools import partial
 
 import param
 from bokeh.io import _CommsHandle
 from bokeh.util.notebook import get_comms
+from bokeh.models.widgets import Select, Slider, AutocompleteInput, TextInput
+from bokeh.layouts import layout, gridplot, widgetbox, row, column
 
+from ...core import Store, NdMapping, OrderedDict
+from ...core.util import drop_streams, unique_array, isnumeric, wrap_tuple_streams
 from ..widgets import NdWidget, SelectionWidget, ScrubberWidget
 from .util import serialize_json
 
+
+
+class BokehServerWidgets(param.Parameterized):
```

Comment:
Hmm, I guess I can create the baseclass but it's all kind of pointless, while Renderer is expecting these attributes they're not actually used in the server case.